tests/builders/test_builders.py

```diff
@@ -138,3 +138,14 @@ def test_file_count():
         configuration=config,
     )
     cmd.build()
+
+
+def test_regex_check():
+    """This test buildspec using status check with  'status_regex'"""
+    cmd = BuildTest(
+        buildspecs=[
+            os.path.join(here, "status_regex.yml")],
+        buildtest_system=system,
+        configuration=config,
+    )
+    cmd.build()
```

Comment:
@shahzebsiddiqui  When I tried running coverage, I see the error `There are no config files to process.`. This might be because on my side it says that the symbolic link is invalid. However, @szuananwar says that the symbolic link works on her side and the regresion test works properly. Is there a reason why it works for her but not during the regresion test by github?